model/task_lifecycle.go

```diff
@@ -1301,7 +1331,9 @@ func updateVersionStatus(v *Version) (string, error) {
 	return versionStatus, nil
 }
 
-func UpdatePatchStatus(p *patch.Patch, versionStatus, buildVariant string) error {
+// UpdatePatchStatus updates the status of a patch. For PR patches, the build associated
+// with the passed in buildVariant has its GitHub status updated.
```

Comment:
It seems like it doesn't update the PR patch anymore. Should the second sentence be for `UpdateVersionAndPatchStatusForBuilds`?

Reply:
Yes, moved to `UpdateVersionAndPatchStatusForBuilds` and updated the wording.